webapp/src/shared/components/Avatar/Avatar.tsx

```diff
@@ -7,6 +7,7 @@ import { Link } from 'react-router-dom';
 
 export type AvatarProps = {
   url: string;
+  imgHash?: number /* TODO: fix auto reloads when avatar is updated, possibly with this */;
```

Comment:
When we update the avatar, your browser's cache will probably not update all other instances of avatars in the parent pages, unless you force a reload. This should be improved.